### PR TITLE
Add prompb README

### PR DESCRIPTION
**Pull request by henridf**
Just a small doc on requirements to compile the proto defs.

### CODE REVIEW

File: prompb/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+The compiled protobufs are version controlled and you won't normally need to
+re-compile them when building Prometheus.
+
+If however you have modified the defs and do need to re-compile, run
+`./scripts/genproto.sh` from the parent dir.
+
+In order for the script to run, you'll need `protoc` (version 3.5) in your
+PATH, and the following Go packages installed:
```
Comment (brian-brazil) [inline prompb/README.md]: It seems a bit odd to me that we'd need to list transitive dependencies like this. Is there one particular Go package that's required that can be go get-ed?

Comment (henridf) [reply prompb/README.md]: you're right; I simplified the list

Comment (brian-brazil) [reply prompb/README.md]: I was thinking more that there should only be one dependency we need, it feels odd that there's 4 (and I didn't need 4 the last time I regenerated this).

Comment (henridf) [reply prompb/README.md]: If you're thinking that we shouldn't need all of these packages, I'm pretty sure that's the case. It took a bit of fumbling about to figure out what needed to be imported and that's why I figured this README would be useful; 

But maybe I'm missing something altogether and there's a simpler way - happy to drop this PR if so.

Comment (brian-brazil) [reply prompb/README.md]: The PR is welcome, I'm just wondering if we can make it a bit simpler. Let me have a poke.

Comment (brian-brazil) [reply prompb/README.md]: Okay, looks like all those are required and github.com/gogo/protobuf too

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**